index.html

```diff
@@ -1,41 +1,81 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width" />
-    <title>Open Source Ideas</title>
-
-    <!-- css -->
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.7/css/materialize.min.css">
-    <link rel="stylesheet" href="css/style.css">
-  </head>
-  <body>
-    <div class='navbar-fixed'>
-      <nav class='teal lighten-1'>
-        <div class="nav-wrapper">
-          <div class="container">
-            <a href="#" class="brand-logo">
-              Open Source Ideas
-            </a>
-          </div>
-        </div>
-        <div class="progress">
-          <div class="indeterminate"></div>
-        </div>
-      </nav>
-    </div>
+<title>Open Source Ideas</title>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<meta name="description" content="Ever had a cool idea for an Open Source project but didn't have the time to implement it yourself? Let someone else give it a try! ">
+<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
```

Comment:
The stylesheet referenced has _177_ uses of `!important`, which is really something you wish to avoid. It makes it quite hard to work with. While W3Schools has some decent content, a lot of it is sadly like this.

I would recommend looking into more mature and _better_ frameworks such as materializecss, tailwind, bootstrap, etc.